bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WWWJets_4f/WWWJets_4f_NLO_proc_card.dat

```diff
@@ -0,0 +1,14 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model loop_sm-no_b_mass
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
```

Comment:
since finally you are using 4f, you can remove those lines and use "import model loop_sm" directly